.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: gomod
```

Comment:
just curious if dependabot will also update vendor dir or someone needs to run it manually ? :thinking:

Reply:
It seems it will: https://github.blog/changelog/2020-10-19-dependabot-go-mod-tidy-and-vendor-support/

Reply:
nice, I didn't know that it has above feature recently :100: